### ISSUE TITLE
Dark Look And Feel Theme issue

### ISSUE THREAD

**Issue by Namphibian** [jGauravGupta/JPAModeler#159]
Hi All.

Apologies for extended radio silence from myself. I was ill due to buckets of stress. I am back and want to pick up with this modeller.

Background:

I am just using a normal maven web project and cannot add the modeller. Running Java 8, netbeans 8.2. When trying to add new JPA model to the project the plugin crashes. I suspect I cannot just add a new model to a Maven project however since I have no database connection or classes defined yet and would like to use the plug in to drive **a top down approach i.e. model before I code** thus I choose the J**PA model vs a new JPA diagram.** 

See screenshot below:

1. I add JPA modeller.

![image](https://cloud.githubusercontent.com/assets/9838598/23777539/9d57b09c-0589-11e7-8fcc-76549f01bba8.png)

2. I choose the folder and give it a name( i choose the src folder)
![image](https://cloud.githubusercontent.com/assets/9838598/23777566/c37df966-0589-11e7-87bc-9ccb91748cc1.png)

3. I get the evil chameleon laughing at me.....

![image](https://cloud.githubusercontent.com/assets/9838598/23777577/db649d6e-0589-11e7-8191-10e71326bdd2.png)

Some notes:
1. If I click on the project node and add the JPA modeller like above it does not create the Filename.jpa on the project node list. See screenshot below:
![image](https://cloud.githubusercontent.com/assets/9838598/23777650/3f4a75a6-058a-11e7-9352-442724dcc9bd.png)

2. If I click  on the source packages node I still get the exception but I do have a  Filename.jpa listed.
![image](https://cloud.githubusercontent.com/assets/9838598/23777665/564995b6-058a-11e7-9bec-b44f05224377.png)

In both cases I am now blocked and cannot proceed any further. I keep on getting exceptions

My Pom file looks like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>com.mycompany</groupId>
    <artifactId>JPATestMe</artifactId>
    <version>1.0-SNAPSHOT</version>
    <packaging>war</packaging>

    <name>JPATestMe</name>

    <properties>
        <endorsed.dir>${project.build.directory}/endorsed</endorsed.dir>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    </properties>
    
    <dependencies>
        <dependency>
            <groupId>org.primefaces</groupId>
            <artifactId>primefaces</artifactId>
            <version>5.0</version>
        </dependency>
        <dependency>
            <groupId>javax</groupId>
            <artifactId>javaee-web-api</artifactId>
            <version>7.0</version>
            <scope>provided</scope>
        </dependency>
    </dependencies>
    <build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.1</version>
                <configuration>
                    <source>1.7</source>
                    <target>1.7</target>
                    <compilerArguments>
                        <endorseddirs>${endorsed.dir}</endorseddirs>
                    </compilerArguments>
                </configuration>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-war-plugin</artifactId>
                <version>2.3</version>
                <configuration>
                    <failOnMissingWebXml>false</failOnMissingWebXml>
                </configuration>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-dependency-plugin</artifactId>
                <version>2.6</version>
                <executions>
                    <execution>
                        <phase>validate</phase>
                        <goals>
                            <goal>copy</goal>
                        </goals>
                        <configuration>
                            <outputDirectory>${endorsed.dir}</outputDirectory>
                            <silent>true</silent>
                            <artifactItems>
                                <artifactItem>
                                    <groupId>javax</groupId>
                                    <artifactId>javaee-endorsed-api</artifactId>
                                    <version>7.0</version>
                                    <type>jar</type>
                                </artifactItem>
                            </artifactItems>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
        </plugins>
    </build>
    <repositories>
        <repository>
            <url>http://repository.primefaces.org/</url>
            <id>PrimeFaces-maven-lib</id>
            <layout>default</layout>
            <name>Repository for library PrimeFaces-maven-lib</name>
        </repository>
    </repositories>
</project>
```

When I try to add it I get the following exception:

```

Message : org.netbeans.modeler.scene.vmd.AbstractPModelerScene.init(AbstractPModelerScene.java:150)

StackTrace : 
java.lang.NullPointerException
	at org.netbeans.modeler.scene.vmd.AbstractPModelerScene.init(AbstractPModelerScene.java:150)
	at org.netbeans.jpa.modeler.specification.model.scene.JPAModelerScene.init(JPAModelerScene.java:345)
	at org.netbeans.modeler.specification.model.file.action.ModelerFileActionListener.openModelerFile(ModelerFileActionListener.java:150)
	at org.netbeans.modeler.specification.model.file.action.ModelerFileActionListener.openModelerFile(ModelerFileActionListener.java:77)
	at org.netbeans.modeler.specification.model.file.action.ModelerFileActionListener.openModelerFile(ModelerFileActionListener.java:65)
	at org.netbeans.modeler.specification.model.file.action.ModelerFileActionListener.lambda$actionPerformed$0(ModelerFileActionListener.java:60)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:109)
	at java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:184)
	at java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:229)
	at java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:227)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:227)
	at java.awt.Dialog.show(Dialog.java:1084)
	at org.netbeans.core.windows.services.NbPresenter.superShow(NbPresenter.java:1086)
	at org.netbeans.core.windows.services.NbPresenter.doShow(NbPresenter.java:1136)
	at org.netbeans.core.windows.services.NbPresenter.run(NbPresenter.java:1108)
	at org.netbeans.core.windows.services.NbPresenter.run(NbPresenter.java:131)
	at org.netbeans.modules.openide.util.NbMutexEventProvider$Event.doEventAccess(NbMutexEventProvider.java:138)
	at org.netbeans.modules.openide.util.NbMutexEventProvider$Event.readAccess(NbMutexEventProvider.java:98)
	at org.netbeans.modules.openide.util.LazyMutexImplementation.readAccess(LazyMutexImplementation.java:94)
	at org.openide.util.Mutex.readAccess(Mutex.java:218)
	at org.netbeans.core.windows.services.NbPresenter.show(NbPresenter.java:1093)
	at java.awt.Component.show(Component.java:1673)
	at java.awt.Component.setVisible(Component.java:1625)
	at java.awt.Window.setVisible(Window.java:1014)
	at java.awt.Dialog.setVisible(Dialog.java:1005)
	at org.netbeans.core.windows.services.DialogDisplayerImpl$1AWTQuery.showDialog(DialogDisplayerImpl.java:266)
	at org.netbeans.core.windows.services.DialogDisplayerImpl$1AWTQuery.run(DialogDisplayerImpl.java:192)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)


ModelerFile : 
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<jpa:entity-mappings v="3.5.1" rws="_14891072107454" cws="_14891072107454" sm="false" xs="false" id="_14891072107453" xsi:schemaLocation="http://java.sun.com/xml/ns/persistence/orm orm_2_1.xsd" xmlns:java="http://jcp.org/en/jsr/detail?id=270" xmlns:jpa="http://java.sun.com/xml/ns/persistence/orm" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <jpa:snp loc="BEFORE_PACKAGE" e="true">/**
 * This file was generated by the JPA Modeler
 */</jpa:snp>
    <jpa:wsl>
        <jpa:ws n="Main" id="_14891072107454"/>
    </jpa:wsl>
</jpa:entity-mappings>
```
```

**Comment by jGauravGupta** [comment]
Please uninstall the dark theme and check again .

**Comment by Namphibian** [comment]
Confirmed Darcula theme caused problem. Non bug. PICNIC(Problem In Chair Not In Computer) error.

**Comment by jGauravGupta** [comment]
Yes but need to fix :)

**Comment by jGauravGupta** [comment]
REF  #93

**Comment by Namphibian** [comment]
Thank you. I prefer the darker themes its easier on my eyes. However I can still work on any theme so would make it a low priority.

**Comment by jGauravGupta** [comment]
This issue is not reproducible in my development environment and reported by any many dark theme users, It would be really great help , if you can help by fixing it.

**Comment by jGauravGupta** [comment]
Issue captured fix will be available in next release v3.6

**Comment by jGauravGupta** [comment]
Please install v3.6 and confirm the issue.
